Makefile

```diff
@@ -77,11 +77,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build:
-ifeq ($(BUILDER),buildah)
-	$(BUILDER) -t ${IMAGE} bud .
-else
-	$(BUILDER) build . -t ${IMAGE}
-endif
+	cekit -v build $(BUILDER)
```

Comment:
how do you pass the `IMAGE` information there ?

Reply:
It is defined in the image descriptor [here](https://github.com/kiegroup/kogito-cloud-operator/pull/784/files#diff-eda18bf907bf5619e4d297d9a06cd67ea32859f6fbff3199904d6e238091788cR12)
I added its entry in the `bump-version.sh` so it is changed at our release time :)

Reply:
yeah but we have this setup: https://github.com/kiegroup/kogito-cloud-operator/blob/master/Makefile#L20
which means that if I set the `IMAGE` var, I would expect the build to create the image with that name. maybe adding a docker tag in the `docker-build` make command ?
moreover, the push would then failed also.

Reply:
I am not sure how can I define the image tag from cekit at runtime, let try some things with runtime override and update here

Reply:
I raised this in `CEKit` channel and looks like it won't be possible for now. I raised an [issue](https://github.com/cekit/cekit/issues/689) as advied by there team. So for now I went ahead with the `tag` approach

Reply:
we can use --overrides to do it, e.g.:

```
      Please note that order matters; overrides on the right hand side take precedence! For
      example:

          $ cekit build --overrides '{"from": "custom/image:1.0"}' --overrides '{"from":
          "custom/image:2.0"}'

```

Reply:
something like 
```cekit -v build $(BUILDER) --overrides '{"version": "custom-tag"}'```

does it work for this use case?

another option as well would be tag the images manually after build.

Reply:
> something like
> `cekit -v build $(BUILDER) --overrides '{"version": "custom-tag"}'`
> 
> does it work for this use case?
> 
No it won't be it'll only change the version but we want the whole image name to change if IMAGE is given by developer I assume it is because he would want the image to be pushed in his namespace on quay. But seems it is not possible in cekit with the multi-stage build right now.

> another option as well would be tag the images manually after build.

Yeah, that's what I have done right now in the `make docker-build` target we build the image and then tag it with `${IMAGE}`.

Reply:
Ok @Kaitou786 , thanks.